src/main/java/co/inventrosoft/springboottask/repository/TeamRepositoryFileImpl.java

```diff
@@ -0,0 +1,72 @@
+package co.inventrosoft.springboottask.repository;
+
+
+import co.inventrosoft.springboottask.model.Team;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+
+@Repository
+public class TeamRepositoryFileImpl implements TeamRepository{
+    private final static String teamsFile = "teams.json";
```

Comment:
1) Order should not be changed.
`public static final String ...`

2) Constant`s name should be in upper case.